### ISSUE TITLE
ALT+Backspace does not undo - PC style

### ISSUE THREAD

**Issue by theflashback** [tekezo/Karabiner#331]
Folks,

I have trouble undoing commands similar to Ctrl+Z using Microsoft keyboard on OSX Yosemite. Can you please fix it?

Thanks,

Serge

**Comment by tekezo** [comment]
I've added new setting at v10.6.1.
Please upgrade to the latest version.

How to upgrade to the latest beta version:
https://pqrs.org/osx/karabiner/document.html#check-for-updates

![150121-0001](https://cloud.githubusercontent.com/assets/659178/5823101/64ec420e-a11d-11e4-8f28-ff51388c4da9.png)

**Comment by theflashback** [comment]
Folks, I am on the latest version but I don't seem to find this setting (only Ctrl+Z is enabled).  Let me know how to get hands on it.
Thanks!

**Comment by tekezo** [comment]
Please confirm your Karabiner version in Misc & Uninstall.
Are you using v10.6.1 or later?

**Comment by theflashback** [comment]
I am on 10.6.1

SER



On Feb 23, 2015, at 02:09, Takayama Fumihiko <notifications@github.com<mailto:notifications@github.com>> wrote:


Please confirm your Karabiner version in Misc & Uninstall.
Are you using v10.6.1 or later?

**Comment by tekezo** [comment]
Hmm, please show me a result of searching "pc style undo" in Preferences.

**Comment by theflashback** [comment]
Let me try again in 8 hours - I am in California in is 2am - I have a lot of people in my group that were forced to use Macs due to iOS development. We are all senior Windows people and Mac keyboard mappings as well as Xcode itself drive us insane :-)

I can also donate for future dev to keep this project running!

Thanks!!

Serge



On Feb 23, 2015, at 02:31, Takayama Fumihiko <notifications@github.com<mailto:notifications@github.com>> wrote:


Hmm, please show me a result of searching "pc style undo" in Preferences.

**Comment by theflashback** [comment]
Attached.

________________________________

**Comment by tekezo** [comment]
The attached file is missing. Please resend.

**Comment by theflashback** [comment]
________________________________

**Comment by theflashback** [comment]
Fumihiko,

I am on 10.6.0 - how do I upgrade to 10.6.1 ?

-SER

________________________________

**Comment by tekezo** [comment]
How to upgrade to the latest beta version:
https://pqrs.org/osx/karabiner/document.html#check-for-updates

**Comment by theflashback** [comment]
Funihiko,

I tried that - it keeps saying no updates found. I am on the 10.6.0

Can you send me the link to where I can download the latest from?

Serge



On Feb 24, 2015, at 21:22, Takayama Fumihiko <notifications@github.com<mailto:notifications@github.com>> wrote:


How to upgrade to the latest beta version:
https://pqrs.org/osx/karabiner/document.html#check-for-updates

**Comment by tekezo** [comment]
Did you press "check for updates including beta version" button?
![using-beta](https://cloud.githubusercontent.com/assets/659178/6384129/8ffc7032-bd9b-11e4-9d3c-39091d188738.png)

**Comment by theflashback** [comment]
Done! thanks!!
________________________________
